ui/nuxeo-document-permissions/nuxeo-popup-permission.js

```diff
@@ -412,7 +413,7 @@ import '../widgets/nuxeo-user-suggestion.js';
         users: [],
         username: null,
         email: null,
-        permission: 'Read',
+        permission: null,
```

Comment:
Be conscious that this works because https://github.com/nuxeo/nuxeo-elements/blob/master/ui/widgets/nuxeo-select.js#L249-L252